packages/core/src/signer/signer/index.ts

```diff
@@ -118,6 +120,13 @@ export abstract class Signer {
           signature.signature,
           signature.identity,
         );
+
+      case SignerSignType.REI:
```

Comment:
Because in the verifyMessageCkbSecp256k1 method, there is a preset prefix, it is not possible to implement the solution where the Rei wallet sets its own prefix.

Reply:
The prefix is from the Neuron wallet
https://github.com/nervosnetwork/neuron/blob/e9eab5ee3851106181675e4adc4a19b2b1fad044/packages/neuron-wallet/src/services/sign-message.ts#L11
and is considered as the standard among the Nervos ecosystem. If there is no special reason for a new prefix, I recommend using the same for compatibility with other ecosystem wallets.

Reply:
![image](https://github.com/user-attachments/assets/752c12c4-3376-4e1b-940d-8396fc5b10af)
![image](https://github.com/user-attachments/assets/ef71706d-3a88-4168-b9ed-a4ac58652ef4)
I would like to ask for your advice. If I use Rei's own prefix, I need to add a new SignerSignType (SignerSignType.REI). If I use the built-in CkbSecp256k1 type, I will need to add the prefix "Nervos Message".

Reply:
> If I use the built-in CkbSecp256k1 type, I will need to add the prefix "Nervos Message".

Sorry for the misunderstanding.

My suggestion is: It's better to force the prefix "Nervos Messsage:" **inside the implementation of the Rei Wallet**. CCC is not running in a standalone environment. Scam apps might modify CCC's behaviour to misguide users into signing dangerous messages like transactions (hash).

To avoid the abovementioned situation, we should **modify the implementation of the Rei Wallet** to add the prefix, which can not be tampered with since it's added outside of the app environment.